gov2/dynamodb/README.md

```diff
@@ -1,113 +1,94 @@
-# AWS SDK for Go V2 code examples for Amazon DynamoDB
+# DynamoDB code examples for the AWS SDK for Go
```

Comment:
AWS SDK for Go --> SDK for Go 
(OK to use short SDK names in titles)

Reply:
done